src/shared/utils/app/setup-date-fns.ts

```diff
@@ -5,7 +5,7 @@ const EN_US = "en-US";
 
 export default async function () {
   let lang = I18NextService.i18n.language;
-  if (lang === "en") {
+  if (lang === "en" || lang === "en-AU") {
```

Comment:
Was datefns having trouble with `en-AU`?

Reply:
Yeah, it wasn't working in my browser without this... Can remove it, it might have just been my machine... I can't seem to start `lemmy-ui` anymore either. I'll have another look tomorrow. 😖
![image](https://github.com/LemmyNet/lemmy-ui/assets/293107/962fecc9-fe81-4266-978b-46030e43a8d2)

Reply:
I managed to get it working by changing the build:dev command, and this `en-AU` doesn't seem to make any difference anymore, weird.  I'll revert it back 😇